### PR TITLE
Standardize documents descriptions

### LINKED ISSUE #1188: Standarize Documents descriptions
The description of `Planning/Tender/Award/Contract/Contract.Implementation.Documents` is not standard across blocks, see below. 

I would suggest:
- Standardizing the phrasing, e.g. "Documents related to the [block name] (for example [block-related examples]." Glad for second opinions on whether the descriptions should start with "All" and whether we need the examples.
- The standardization consists mainly of:
    - Removing "attachment" (not used anywhere else in the schema, covered by "document") and "reports" (covered by "document").
    - Removing "A list" (We typically don't mention that arrays are lists, even though, beyond documents, this is also done for additional identifiers, milestones and transactions and should perhaps be removed there as well?)
    - "Including any notices" - can be mentioned in the examples section

#### Overview

`Planning.Documents`
_A list of documents related to the planning process._ 

`Tender.Documents`
_All documents and attachments related to the tender, including any notices. See the documentType codelist for details of potential documents to include. Common documents include official legal notices of tender, technical specifications, evaluation criteria, and, as a tender process progresses, clarifications and replies to queries._

 `Award.Documents`
_All documents and attachments related to the award, including any notices._ 

 `Contract.Documents`, 
_All documents and attachments related to the contract, including any notices._

 `Contract.Implementation.Documents`, 
_Documents and reports that are part of the implementation phase e.g. audit and evaluation reports._

### PR DESCRIPTION
**Pull request by JachymHercher**
closes #1188

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 * [#1218](https://github.com/open-contracting/standard/pull/1218) Add the 'COFOG' code from the organization classifiation extension to the `itemClassificationScheme` codelist.
 
 ### Schema
-
+* [#1229](https://github.com/open-contracting/standard/pull/1229) Standardize the descriptions of `Planning.Documents`, `Tender.Documents`, `Award.Documents`, `Contract.Documents`, and `Contract.Implementation.Documents`.
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
* [#1229](https://github.com/open-contracting/standard/pull/1229) Standardize the descriptions of `planning.documents`, `tender.documents`, `awards.documents`, `contracts.documents`, and `contracts.implementation.documents`.
```

File: schema/dereferenced-release-schema.json [hunk 2]
```diff
@@ -1012,7 +1012,7 @@
         },
         "documents": {
           "title": "Documents",
-          "description": "A list of documents related to the planning process.",
+          "description": "Documents related to the planning stage (for example notices, needs assessments and market studies).",
```
Comment (jpmckinney) [inline schema/dereferenced-release-schema.json]: Add a comma after "for example" in all cases.

```suggestion
          "description": "Documents related to the planning stage (for example, notices, needs assessments and market studies).",
```

File: schema/dereferenced-release-schema.json [hunk 3]
```diff
@@ -5304,7 +5304,7 @@
           },
           "documents": {
             "title": "Documents",
-            "description": "All documents and attachments related to the award, including any notices.",
+            "description": "Documents related to the award stage (for example notices, bids, evaluation reports).",
```
Comment (jpmckinney) [inline schema/dereferenced-release-schema.json]: For award, contract and implementation, the documents relate to the specific award, contract, and contract implementation – not the stage as a whole.

**Review by jpmckinney — COMMENTED** [review]
I commented on the dereferenced schema, but I meant the release schema.

File: docs/history/changelog.md (hunk 1 above)
Comment (JachymHercher) [reply docs/history/changelog.md]: Shouldn't it be award and contract, not awards and contracts? (Sorry for the capitalization.)

File: schema/dereferenced-release-schema.json (hunk 3 above)
Comment (JachymHercher) [reply schema/dereferenced-release-schema.json]: Makes sense. I wasn't sure whether implementation should be "implementation", "the implementation" or "implementation of the contract". I've tentatively included it with the last option.

File: schema/dereferenced-release-schema.json (hunk 2 above)
Comment (JachymHercher) [reply schema/dereferenced-release-schema.json]: Added in https://github.com/open-contracting/standard/pull/1229/commits/c351ca4cbb9df2edaaebda33100ede21ee1069db

File: schema/dereferenced-release-schema.json (hunk 3 above)
Comment (JachymHercher) [reply schema/dereferenced-release-schema.json]: Added in https://github.com/open-contracting/standard/pull/1229/commits/c351ca4cbb9df2edaaebda33100ede21ee1069db

File: docs/history/changelog.md (hunk 1 above)
Comment (jpmckinney) [reply docs/history/changelog.md]: The JSON path is plural. The definition in the JSON Schema is singular and capitalized. The data uses the JSON paths, so we prefer the uncapitalized, plural versions.

**Pull request merged**